ConsoleCalculator.Tests/CalculatorFixture.cs

```diff
@@ -1,14 +1,178 @@
-using System;
 using Xunit;
 
 namespace ConsoleCalculator.Tests
 {
     public class CalculatorFixture
     {
+        Calculator calculator = new Calculator();
+
+        private string SendMultipleKey(char[] key)
+        {
+            var i = 0;
+            for (i = 0; i < key.Length - 1; i++)
+                calculator.SendKeyPress(key[i]);
+            return calculator.SendKeyPress(key[i]);
+        }
+
+
         [Fact]
-        public void DummyTest()
+        public void TestKeyPress()

```

Comment:
Test case method should be more descriptive like "Single_KeyPress_With_Value_1_Should_Return_1"